src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -747,9 +747,61 @@
 """
 
 helps['sql virtual-cluster show'] = """
-type: command
-short-summary: Get the details for a virtual cluster.
-examples:
-  - name: Get the details for a virtual cluster
-    text: az sql virtual-cluster show -g mygroup -n mycluster
-"""
+    type: command
+    short-summary: Get the details for a virtual cluster.
+    examples:
+        - name: Get the details for a virtual cluster
+          text: az sql virtual-cluster show -g mygroup -n mycluster
```

Comment:
the naming conventions seem to be a bit inconsistent with other azure cli commands. (MyResourceGroup etc.)
but that's minor, just mentioning it. not a showstopper for merging this pr.

Reply:
this cmdlet help is not added my me. I think it was from a merge previously. The code is already in the repo: https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/sql/_help.py

Do you want me to remove it manually?

Reply:
no worries, just a suggestion